src/guides/configuration.md

```diff
@@ -0,0 +1,58 @@
+# Configuration
+
+Glimmer apps work right out of the box, but you can configure them to fit your needs.
+
+## Root Container
+
+Glimmer apps contain an `index.html` file that has the following div, which is where all your markup will reside in:
+
+```html
+<!-- ui/index.html -->
+<div id="app">
+  <!--Your app will render here at runtime-->
+</div>
+```
+
+If you'd like to change the container, open `ui/index.ts` and change
+
+```js
+const containerElement = document.getElementById('app');
+```
+
+to
+
+```js
+const containerElement = document.getElementById('conference-speaker-app');
+```
+
+Your host page can now hold multiple glimmer apps if you'd like. It can even point to other hosted scripts (that you trust) that other teams might host so you can get the latest updates without integration.
```

Comment:
It could always host multiple apps, no? There's no need to change the initial id. I think you're missing instructions on how to actually boot the second components, one needs to update `index.ts`, I think?